build.psm1

```diff
@@ -1167,6 +1167,7 @@ function Start-PSPester {
     [CmdletBinding(DefaultParameterSetName='default')]
     param(
         [Parameter(Position=0)]
+        [ArgumentCompleter({param($c,$p,$word) Get-ChildItem -Recurse -File -LiteralPath $PSScriptRoot/Test -filter *.tests.ps1 | Where-Object FullName -like "*$word*" })]
```

Comment:
Shouldn't this limit the search to those under `powershell` folder which has ps1 tests?

```suggestion
        [ArgumentCompleter({param($c,$p,$word) Get-ChildItem -Recurse -File -LiteralPath $PSScriptRoot/Test/PowerShell -filter *.tests.ps1 | Where-Object FullName -like "*$word*" })]
```